### ISSUE TITLE
Activate the current page in the list

### ISSUE THREAD

**Issue by ArtskydJ** [ArtskydJ/josephdykstra.com#8]
# current behaviour
![am-1](https://cloud.githubusercontent.com/assets/1833684/7404775/2236f150-eeae-11e4-9797-826775688b45.PNG)

# expected behaviour
![am-2](https://cloud.githubusercontent.com/assets/1833684/7404776/223968e0-eeae-11e4-8920-320f6eff0f25.PNG)

Consider removing the `Pages` header, or making it orange:
```html
<div class="panel panel-warning">
  <div class="panel-heading"> <h3 class="panel-title">Panel warning</h3> </div>
  <div class="panel-body"> Panel content </div>
</div>
```

**Comment by ArtskydJ** [comment]
Removed `Pages` header...

**Comment by ArtskydJ** [comment]
Fixed in 3611db61620310060690e1a1294f3f96d21f22c4.
